default_bd.cpp

```diff
@@ -79,7 +79,7 @@ mbed::BlockDevice* get_secondary_bd(void) {
     // slice it into a smaller size, 1MB at the beginning
 
     mbed::BlockDevice* default_bd = mbed::BlockDevice::get_default_instance();
-    static mbed::SlicingBlockDevice sliced_bd(default_bd, 0x0, 0xDD000);
+    static mbed::SlicingBlockDevice sliced_bd(default_bd, 0x0, MCUBOOT_SLOT_SIZE);
```

Comment:
It's worth updating the comment just above to remove the explicit size of the first slot.

Reply:
Done